src/ui/src/app/chart-details/chart-details.component.html

```diff
@@ -1,16 +1,37 @@
-<app-header-bar></app-header-bar>
+<app-header-bar [transparent]=true></app-header-bar>
 <div>
   <app-panel *ngIf="!chart && !loading" class="app-panel--container">
     <h2>Sorry, we couldn't find the chart</h2>
   </app-panel>
   <app-loader [loading]="loading">
     <section class="chart-details" *ngIf="chart">
-      <article class="chart-details__docs">
-        <app-chart-details-readme [chart]=chart [currentVersion]=currentVersion></app-chart-details-readme>
-      </article>
-      <aside class="chart-details__info">
-        <app-chart-details-info [chart]=chart [currentVersion]=currentVersion></app-chart-details-info>
-      </aside>
+      <div class="chart-details__header">
+        <div class="chart-details__header__background" [style.background-color]="chartColor"></div>
```

Comment:
I like this concept but we don't control the color of icons. If the "main" color of an icon is to bright or dark, the readability of the content will be affected. Also, if the color is too saturated, the general design will look weird for that page.